blogs/scripts/taxonomy.js

```diff
@@ -1,83 +1,58 @@
 import { toClassName } from './lib-franklin.js';
 
 /**
- * Find the second-level list items and put them into a single array.
- * @returns {Array} An array of tag strings
+ * Retrieve tags from API.
+ * @param {String} [category] The category of tags to be returned (keysight|segmentation)
+ * @param {String} [lang] The language of the tags to be returned
+ * @returns {Array} An array of tag objects
  */
-async function getTags(ul) {
-  const tagList = ul.querySelectorAll('ul li ul li');
-  const tagArray = [...tagList];
-  const textArray = tagArray.map((li) => toClassName(li.textContent));
-  return textArray;
-}
-
-/**
- * Retrieve HTML from tags.docx.
- * @returns {Array} An array of tag strings
- */
-async function getFranklinTags() {
+async function getAEMTags(category = 'keysight', lang = 'en') {
```

Comment:
love the use of default parameters here

Reply:
I do think this function probably will need to be exported to fulfill the tagger needs